gqlgen.yml

```diff
@@ -1,11 +1,12 @@
 # Where are all the schema files located? globs are supported eg  src/**/*.graphql
 schema:
-  - serve/graph/*.graphql
+  - serve/graph/schema/*.graphql
+  - serve/graph/schema/**/*.graphql
```

Comment:
As I had more filters and schema types, I realized it would be nice to separate them.

So I added a subdirectory to get the schema's subdirectories.